asyn/drvAsynSerial/drvAsynIPPort.c

```diff
@@ -693,6 +699,7 @@ drvAsynIPPortConfigure(const char *portName,
     asynStatus status;
     char *cp;
     int nbytes;
+    int autoConnect = noAutoConnect & USERFLAG_NO_AUTO_CONNECT ? 0 : 1;
```

Comment:
I would suggest renaming the `int noAutoConnect` argument to the `drvAsynIPPortConfigure()` function (and change its name in the documentation too) to avoid future confusion. The obvious name would be `userFlags`. You should probably also check that the value doesn't have any unused bits set — if the code doesn't object to them now, it'll break IOC applications that might set them by mistake or start using them for other things when the next bit in userFlags gets allocated. Also don't use a conditional operator when the logical meaning is 'not':
``` diff
-int autoConnect = noAutoConnect & USERFLAG_NO_AUTO_CONNECT ? 0 : 1;
+int autoConnect = ! (userFlags & USERFLAG_NO_AUTO_CONNECT);
```